src/analysis/normed_space/multilinear.lean

```diff
@@ -245,6 +245,42 @@ def mk_continuous (C : ℝ) (H : ∀ m, ∥f m∥ ≤ C * univ.prod (λi, ∥m i
   continuous_multilinear_map 𝕜 E₁ E₂ :=
 { cont := f.continuous_of_bound C H, ..f }
 
+lemma restr_norm_le {k n : ℕ} (f : multilinear_map 𝕜 (λ i : fin n, G) E₂)
```

Comment:
I have a hard time understanding what this lemma says. I think it deserves a docstring. Also, can the `@multilinear_map.restr` be done without `@` if some argument becomes explicit, or if you add some type annotation?

Reply:
I have tried several versions to get rid of the underscores. If I give all types explicitely, then it is fine, but this is clearly overkill: normally, Lean should be able to figure out everything by itself. In fact, if you use multilinear maps over a ring, then you can use the regular version (without the `@`), and it is only when you use multilinear maps over a field that this becomes a problem (I don't know why). In this file, I also define the restriction of continuous multilinear maps over fields, and then it works prefectly fine, no need to use `@`. And I think it is more natural to keep the same interface for `multilinear_maps.restr` and `continuous_multilinear_maps.restr`.

TLDR: I put a lot of thought in this, and I decided it was better to keep it like that.

Reply:
Docstring added.

Reply:
I have used the elaboration trick mentionned by @gebner on Zulip to get rid of the underscores. This works perfectly, thanks!